api/resolvers/wallet.js

```diff
@@ -126,7 +126,11 @@ export async function getWithdrawl (parent, { id }, { me, models, lnd }) {
     },
     include: {
       user: true,
-      invoiceForward: true
+      invoiceForward: {
+        include: {
+          invoice: true
+        }
+      }
```

Comment:
we need this to fetch the comment of the invoice to display it in the notification page